libs/sr/ui/src/Relic/RelicInventory.tsx

```diff
@@ -0,0 +1,103 @@
+import { useMediaQueryUp } from '@genshin-optimizer/common/react-util'
+import { useInfScroll } from '@genshin-optimizer/common/ui'
+import { ExpandMore } from '@mui/icons-material'
+import {
+  Accordion,
+  AccordionSummary,
+  Box,
+  CardHeader,
+  Grid,
+  Skeleton,
+  Typography,
+} from '@mui/material'
+import { Suspense, useMemo } from 'react'
+import { useDatabaseContext } from '../Context'
+import { RelicCard } from './RelicCard'
+
+const columns = { xs: 1, sm: 2, md: 3, lg: 3, xl: 4 }
+const numToShowMap = { xs: 10, sm: 12, md: 24, lg: 24, xl: 24 }
+
+export function RelicInventory() {
+  const { database } = useDatabaseContext()
+
+  const { relicsIds, totalRelicsNum } = useMemo(() => {
+    const relicsIds = database.relics.keys
+    const totalRelicsNum = relicsIds.length
+    return { relicsIds, totalRelicsNum }
+  }, [database])
+
+  const brPt = useMediaQueryUp()
+  const totalShowing =
+    relicsIds.length !== totalRelicsNum
+      ? `${relicsIds.length}/${totalRelicsNum}`
+      : totalRelicsNum
+  const { numShow, setTriggerElement } = useInfScroll(
+    numToShowMap[brPt],
+    relicsIds.length
+  )
+
+  const relicsIdsToShow = useMemo(
+    () => relicsIds.slice(0, numShow),
+    [relicsIds, numShow]
+  )
+  const showingTextProps = {
+    numShowing: relicsIdsToShow.length,
+    totalShowing: totalShowing,
+  }
+
+  return (
+    <Accordion>
```

Comment:
Also this accordion is full width for some reason
![image](https://github.com/frzyc/genshin-optimizer/assets/36019388/f27f2ea1-63a2-4d7a-9463-ca399abfe475)

Reply:
fixed on ba2186105d0f9a6fc46112936bd0b5dda574e2c7

Reply:
Any suggestion on how to fix this issue? @nguyentvan7

Reply:
fixed on 3a3a79cfa8aecc6d8a03fffe8ef070ab99b731a3